utils/account-setup/index.js

```diff
@@ -0,0 +1,139 @@
+            }
+
+            console.log(`Created ${item} Account`);
+
+            return { "name" : item, "xrplAcc": acc };
+            
+          });
+
+        const newAccounts = await Promise.all(unresolved);
+        // END - Account Creation
+
+        // 5 second wait in order to sync the accounts
+        await new Promise(r => setTimeout(r, 5000));
+        
+        // BEGIN - Trust Lines initiation
+        const foundation_lines = await newAccounts[1].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (foundation_lines.length === 0) {
+            await newAccounts[1].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, TOTAL_MINTED_EVRS);
+        }
+
+        const comm_bank_lines = await newAccounts[2].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (comm_bank_lines.length === 0) {
+            await newAccounts[2].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, PURCHASER_PROGRAM_EVRS);
+        }
+
+        const registry_lines = await newAccounts[3].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (registry_lines.length === 0) {
+            await newAccounts[3].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, TOTAL_MINTED_EVRS);
+        }
+
+        const purchaser_lines = await newAccounts[4].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (purchaser_lines.length === 0) {
+            await newAccounts[4].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, PURCHASER_PROGRAM_EVRS);
+        }
+
+        console.log("Trust Lines initiated");
+        // END - Trust Lines initiation
+
+        // BEGIN - Transfer Currency
+        await newAccounts[0].xrplAcc.makePayment(newAccounts[1].xrplAcc.address, TOTAL_MINTED_EVRS, evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        console.log(`${TOTAL_MINTED_EVRS}/- EVR amount was issued to EVERNODE Foundation`);
+
+        await newAccounts[1].xrplAcc.makePayment(newAccounts[2].xrplAcc.address, PURCHASER_PROGRAM_EVRS, evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        console.log(`${PURCHASER_PROGRAM_EVRS}/- EVR amount was transfered to Community Contract Bank Account by the Foundation`);
+        // END - Transfer Currency
+
+        // ISSUER Blackholing	
+        await newAccounts[0].xrplAcc.setRegularKey(BLACKHOLE_ADDRESS);
+        console.log("Blackholed ISSUER");
+
+        // BEGIN - Log Account Details
+        console.log('\nAccount Details -------------------------------------------------------');
+
+        newAccounts.forEach(element => {            
+            console.log(`Account name :${element.name}`);
+            console.log(`Address : ${element.xrplAcc.address}`);
```

Comment:
Don't we require account secret as well?

Reply:
In terms of safety, I don't think the raw private key should be disclosed as it may expose the user's private key to unauthorized viewers.